snippets/card-product.liquid

```diff
@@ -326,9 +331,14 @@
           {%- elsif card_product.compare_at_price > card_product.price and card_product.available -%}
             <span
               id="Badge-{{ section_id }}-{{ card_product.id }}"
-              class="badge badge--bottom-left color-{{ settings.sale_badge_color_scheme }}"
+              class="badge badge--bottom-left {% if card_product.type == 'Wholesale' and customer.tags contains 'wholesale' %}custom-badge{% endif %} color-{{ settings.sale_badge_color_scheme }}"
+              {% if card_product.type == 'Wholesale' and customer.tags contains 'wholesale' %}style="--color-background: {{ settings.custom_badge_color_bg.red }}, {{ settings.custom_badge_color_bg.green }}, {{ settings.custom_badge_color_bg.blue }};"{% endif %}
```

Comment:
remove custom badge colors, use sale badge styles

Reply:
@heythisisnate we need these custom color pickers for badges, Dawn Colors are limited

![image](https://github.com/konnected-io/shopify-dawn/assets/32586764/50c78227-5a5c-4ec6-a233-1725fab78716)

![image](https://github.com/konnected-io/shopify-dawn/assets/32586764/d6e54602-9797-47b4-8386-be28ea63a938)

Reply:
![Uploading image.png…]()

Reply:
I should have replied under here sorry 
![image](https://github.com/konnected-io/shopify-dawn/assets/32586764/cbdcc490-fbc8-43d4-85c3-221491065faa)